src/selection.ts

```diff
@@ -14,6 +17,38 @@ export const getRange = (selection: Selection): Range => {
   return range;
 };
 
+export const cleanSelection = (selection: Selection): Selection => {
+  const anchor = selection.anchorNode;
+  const focus = selection.focusNode;
+
+  if (!anchor || !focus) {
+    return selection;
+  }
+
+  const anchorParent = anchor.parentNode;
+  const focusParent = focus.parentNode;
+  const beginsOnIframe = isIframe(anchor);
+  const endsOnIframe = isIframe(focus);
+  const range = getRange(selection);
+  const isImgAndFirstElement = isImg(range.startContainer);
```

Comment:
how does this indicate `AndFirstElement` being the first element of the page?

Reply:
Because if an element came before it then that would be used as the startContainer.